contracts/HashRegistrarSimplified.sol

```diff
@@ -486,10 +342,12 @@ contract Registrar {
      *
      * @param _hash The node to release
      */
-    function releaseDeed(bytes32 _hash) onlyOwner(_hash) {
-        entry h = _entries[_hash];
+    function releaseDeed(bytes32 _hash) public onlyOwner(_hash) {
+        Entry storage h = _entries[_hash];
         Deed deedContract = h.deed;
-        if (now < h.registrationDate + 1 years && ens.owner(rootNode) == address(this)) throw;
+
+        require(now >= h.registrationDate + 1 years);
```

Comment:
This needs to be a disjunction; we allow releasing if it's been held for a year *or* the registrar has been replaced.